### DISCUSSION TITLE
Add Option to Skip the Red Flag Period in Synced replays

### DISCUSSION THREAD

**Discussion by p-haugh** [f1multiviewer/issue-tracker#253] [Feature requests]
Add a toggle to skip to end of a red flag period for a synced replay (i.e. Japanese GP 2022)

Watched the Japan GP replay this morning and wanted to skip through the red flag period. However, I was not sure how long it was and didn't want to check in case I came across a spoiler, so spent 5 minutes skipping through until I came to the restart

**Comment by f1multiviewer** [comment]
So you missed the ducks!?
![image](https://user-images.githubusercontent.com/106821994/194753577-bb1100cf-ec73-49c4-8a07-b0be21e4396a.png)

**Reply by p-haugh** [reply]
Unfortunately yes 🥲😂
